packages/scandipwa/src/component/Menu/Menu.component.js

```diff
@@ -380,6 +398,7 @@ export class Menu extends PureComponent {
                 <Link to="compare" block="Menu" elem="CompareLink">
                     <CompareIcon />
                     <h4>{ __('Compare products') }</h4>
+                    { `(${this.renderCompareCountInMenu()})` }
```

Comment:
I think you should move braces inside the renderCompareCountInMenu, and here it will be simply `{ this.renderCompareCountInMenu() }`